modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -28,7 +28,7 @@ installation, do not delete these files.
 ====
 
 ifndef::openshift-origin[]
-. Obtain the compressed metal RAW image, `kernel`
+. Obtain the {op-system} ISO image, compressed metal RAW image, `kernel`
```

Comment:
I don't think you need the ISO for PXE boot

Reply:
Yeah, thanks - Chris updated this to `. Obtain the {op-system} `kernel`,
`initramfs` and `rootfs` files`...